src/hooks/useColor.ts

```diff
@@ -70,9 +72,9 @@ export function useColor(token?: Token) {
 
     return () => {
       stale = true
-      setColor('#2172E5')
+      setColor(theme.accent1)
     }
-  }, [src, token])
+  }, [src, theme.accent1, token])
```

Comment:
i guess just on reading this on dark/light switch this will revert the color to accent1 until it re-runs the color get and sets it again, but probably not a big deal and maybe even fast enough to not notice

Reply:
No new query occurs as the token's image's cached and the math's quick to calculate so changing the theme looks instant on the page. Thanks for the callout!